assets/erc-7007/contracts/ERC7007Enumerable.sol

```diff
@@ -1,13 +1,13 @@
 // SPDX-License-Identifier: UNLICENSED
```

Comment:
You'll need to change this to something we can legally redistribute. We prefer CC0-1.0, but I also accept MIT/Apache-2.0/Unlicense if you like.